draggablepanel/src/main/java/com/github/pedrovgs/DraggableView.java

```diff
@@ -52,7 +53,9 @@
   private static final int ONE_HUNDRED = 100;
   private static final float SENSITIVITY = 1f;
   private static final boolean DEFAULT_TOP_VIEW_RESIZE = false;
+  private static final int INVALID_POINTER = -1;
 
+  private int mActivePointerId = INVALID_POINTER;
```

Comment:
This repository doesn't use "m" notation. Please rename this variable to activePointerId.